sopel_modules/chanlogs/__init__.py

```diff
@@ -111,6 +112,17 @@ def _format_template(tpl, bot, trigger, **kwargs):
     return formatted
 
 
+def _channel_is_opted_in(is_channel, channel, opted_in_channels):
+    if is_channel and not opted_in_channels or str(channel) in opted_in_channels: # cast channel to string because I don't trust python
```

Comment:
`channel` should be cast to `sopel.tools.Identifier`, not `str`.

Also PEP8 demands two spaces before `#` to start a line-end comment.

Reply:
Won't everything in the array be a string though?